packages/renderer/src/lib/ui/KubernetesCurrentContextConnectionBadge.spec.ts

```diff
@@ -102,18 +107,44 @@ describe.each<{
     expect(status).toBeNull();
   });
 
-  test('expect badges to show as there is a context', async () => {
-    setState(true);
+  test('expect badges to show as there is a reachable context', async () => {
+    setState({ reachable: true });
     render(KubernetesCurrentContextConnectionBadge);
 
     await vi.waitFor(() => {
       const status = screen.getByRole('status');
       expect(status).toBeInTheDocument();
+      expect(status).toHaveTextContent('Connected');
+    });
+  });
+
+  test('expect badges to show as there is a non reachable context', async () => {
+    setState({ reachable: false });
+    render(KubernetesCurrentContextConnectionBadge);
+
+    await vi.waitFor(() => {
+      const status = screen.getByRole('status');
+      expect(status).toBeInTheDocument();
+      expect(status).toHaveTextContent('Cluster not reachable');
+    });
+  });
+
+  test('expect badges to show as there is an offline context', async () => {
+    if (!experimental) {
+      return;
+    }
```

Comment:
I don't think adding a `testIfExperimental` as shown by the ai-tool is necessary when we can do the following

```ts
test('name', () => {
  // test
}, {
  skip: <condition>
})
```